jenkins-master/jobs/scripts/workspace/requirements.txt

```diff
@@ -1,3 +1,3 @@
 boto==2.38.0
-mozdownload==1.18
+mozinfo==0.8
```

Comment:
This file is only used for the treeherder venv now. So mozdownload is not needed, but mozinfo.